doc/Command-Reference.md

```diff
@@ -7961,8 +7962,292 @@ Go Back To [Beginning of the document](#) or [Beginning of this section](#waterm
 
 ## Software Installation and Management
 
-SONiC software can be installed in two methods, viz, "using sonic-installer tool", "ONIE Installer".
+SONiC software image can be installed in two methods, viz, "using sonic-installer tool", "ONIE Installer".
```

Comment:
Suggest removing the word "software", making a list, and also mentioning other bootloaders. E.g.,

```suggestion
SONiC image can be installed in one of two methods:
1. From within a running SONiC iamge using the `sonic-installer` utility
2. From the vendor's bootloader (E.g., ONIE,  Aboot, etc.)
```